src/App/src/ConfigProvider.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App;
+
+class ConfigProvider
```

Comment:
Add method docblocks to this class, so developers can understand what each method is for.